### DISCUSSION TITLE
UiPath interview questions - xrates

### DISCUSSION THREAD

**Discussion by vajrang-b** [vajrang-b/RPA-Developer-in-30-Days#190] [Show and tell]
![image](https://user-images.githubusercontent.com/5066917/141053752-d9371e76-bc37-4c78-85d9-f4de6b5cff24.png)

**Try solving this project and make pull request**

**Comment by sgsiddalinga** [comment]
I will work on this

**Reply by vajrang-b** [reply]
> Bowser compatibility checked - Yes

**Reply by vajrang-b** [reply]
User Chrome or Internet explorer

**Comment by sgsiddalinga** [comment]
Repetitive -Yes

**Comment by shivamsr7** [comment]
Low Exception Rate- Yes

**Comment by sgsiddalinga** [comment]
Human Intervention Required - No

**Comment by sgsiddalinga** [comment]
Complexity - No

**Reply by vajrang-b** [reply]
Complexity - Low medium or High

**Reply by vajrang-b** [reply]
Low - 5 to 10 steps with 1 or two native applications - time required 1 week to two weeks - two apps with image based automation this becomes high complex process

medium- less than 30 steps  with  2 - 3 apps - time required 2 - 3 weeks
High - more than 3 apps - more than 50 steps - 4 weeks only for development -

**Comment by VenkyValluri** [comment]
Extraction data is in a structured manner

**Comment by sgsiddalinga** [comment]
Risk Involved - No

**Reply by pallavi1208** [reply]
Larg data will be handled - yes

**Comment by vajrang-b** [comment]
Lets proceed with creation of PDD, Process flows, **as is diagram , to be diagram**

**Comment by vajrang-b** [comment]
week 1 - PDD, Process flows, as is diagram, to be diagram, project structure , 
week2 - UAT deployment, testing , final demo
